HTAQ.md

```diff
@@ -0,0 +1,66 @@
+# HOW TO ASK QUESTIONS
+:heavy_exclamation_mark:**PLEASE READ FULLY**:heavy_exclamation_mark:
+## INTRO
+In the world of hackers, the kind of answers you get to your technical questions depends as much on the way you ask the questions as on the difficulty of developing the answer. This guide will teach you how to ask questions in a way more likely to get you a satisfactory answer.
+
+**It sometimes looks like we're reflexively rude to newbies and the ignorant. But this isn't really true. Please try to ask good questions :smile_cat:**
+
+## Before You Ask
```

Comment:
It is a long list 😄 

Let's make it shorter and more concrete and friendly?  As an example:
> If you have a question about course itself - try to use FAQ before.
If you have a question about tasks - good sources will be our chats
If question about JS or HTML/CSS - Web, MDN, etc.

But something more readable.